extension/src/background/index.ts

```diff
@@ -1,5 +1,5 @@
-import { Poller } from "./poller";
 import { browser, Runtime } from "webextension-polyfill-ts";
+import { Poller } from "./poller";
```

Comment:
We should get rid of this file

Reply:
What was it supposed to do ?

Reply:
I think it came with the boilerplate I used